EIPS/eip-7496.md

```diff
@@ -63,12 +66,12 @@ The keys in the `traits` object MUST be unique trait names. If the trait name is
 
 The `displayName` values MUST be unique and MUST NOT collide with the `displayName` of any other traits defined in the metadata.
 
-The `consumptionValidationOnSale` value provides a signal to marketplaces on how to validate the trait value when a token is being sold. If the validation criteria is not met, the sale SHOULD not be permitted by the marketplace contract. If specified, the value of `consumptionValidationOnSale` MUST be one of the following (or it is assumed to be `none`):
+The `validateOnSale` value provides a signal to marketplaces on how to validate the trait value when a token is being sold. If the validation criteria is not met, the sale MUST not be permitted by the marketplace contract. If specified, the value of `validateOnSale` MUST be one of the following (or it is assumed to be `none`):
```

Comment:
I made this `SHOULD` here and below to indicate that it's a best effort practice on the part of the marketplaces and not a strict requirement--maybe another way we can signal that instead? Since these URIs can be generated offchain and there are race conditions, I think it should be clear that this validation isn't something that can be strictly relied on.

Reply:
i think by saying "SHOULD" marketplaces are up to their own discretion to provide the validation or not, even on a case by case basis, and we should make it as required as possible. i will add language that it cannot be relied on though that 's a good point.

Reply:
note added on line 75